stories/icon-tab-bar/icon-tab-bar.stories.js

```diff
@@ -0,0 +1,1434 @@
+export default {
+    title: 'Components/Icon Tab Bar',
+    parameters: {
+        description: `
+The Icon Tab Bar can be used for navigation within an object, or as a filter. Each tab of the component links to a different content area or view.
+
+##Usage
+**Use the icon tab bar if:**
+
+- Your business objects need to show multiple facets at the same time.
+- You want to allow the user to browse through these facets.
+- You need a prominent or very visual filter on top of a list.
+- You have clear-cut process steps that need to be visualized.
+
+**Do not use the icon tab bar if:**
+
+- You plan to use only one single tab.
+
+##Semantic colors
+Only use semantic colors if it is important for users to know that they need to take action (for example, to indicate errors or critical situations requiring action). Otherwise, use the neutral default colors. To add semantic color to a tab use a modifier class from the table below on <code>fd-icon-tab-bar\\_\\_item</code> level.
```

Comment:
Rewrite: Only use semantic colors if it is important for users to know when they need to take action..

Reply:
I didn't change this one.  Replacing `that` with `when` might confuse the user. The user in all cases should take action when the item has semantic color. If we use `when` we might make them think that the color is associated with some time.

Reply:
okay, noted!